types/victory/victory-tests.tsx

```diff
@@ -663,3 +665,151 @@ test = (
         data={[{ name: 'One' }, { name: 'Two' }, { name: 'Three' }]}
     />
 );
+
+const emptyTheme: VictoryThemeDefinition = {};
+
+type RecursiveRequired<T> = Required<
```

Comment:
I wouldn't feed this in through an intermediate type - just use a `-?` modifier.

Also, is this really necessary for the tests?

Reply:
I added this to make sure I had thoroughly tested every property - since many are optional in the actual type, this makes sure the test is thorough

Didn't know about `-?` though.

Reply:
I have switched to `-?`, I like it!

I still maintain that the `RecursiveRequired` is adding value to an otherwise very complicated set of types with mixed optionality.